mmcv/utils/env.py

```diff
@@ -56,19 +56,32 @@ def collect_env():
         if CUDA_HOME is not None and osp.isdir(CUDA_HOME):
             try:
                 nvcc = osp.join(CUDA_HOME, 'bin/nvcc')
-                nvcc = subprocess.check_output(
-                    f'"{nvcc}" -V | tail -n1', shell=True)
+                nvcc = subprocess.check_output(f'"{nvcc}" -V', shell=True)
                 nvcc = nvcc.decode('utf-8').strip()
+                release = nvcc.rfind('Cuda compilation tools')
+                build = nvcc.rfind('Build ')
+                nvcc = nvcc[release:build].strip()
             except subprocess.SubprocessError:
                 nvcc = 'Not Available'
             env_info['NVCC'] = nvcc
 
     try:
-        gcc = subprocess.check_output('gcc --version | head -n1', shell=True)
-        gcc = gcc.decode('utf-8').strip()
-        env_info['GCC'] = gcc
-    except subprocess.CalledProcessError:  # gcc is unavailable
-        env_info['GCC'] = 'n/a'
+        import sysconfig
```

Comment:
That's Awesome, `setuptools._distutils.ccompiler` was able to find my local cl and output its version. But `sysconfig `will return the environment for compiling python, could it get the version of `cl.exe` ? It seems that `ModuleNotFoundError` make CI failed, I'll check it.